server/odc-common/src/main/java/com/oceanbase/odc/common/i18n/I18nOutputSerializer.java

```diff
@@ -16,26 +16,28 @@
 package com.oceanbase.odc.common.i18n;
 
 import java.io.IOException;
-import java.util.HashMap;
 import java.util.Locale;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import org.apache.commons.text.StringSubstitutor;
 import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.stereotype.Component;
 
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
 
+@Component
```

Comment:
avoid involve Bean in common moudle.
create in service layer instead, refer BeanConfiguration .

Reply:
It has been restored and extracts MessageTemplateProcessor#getLocalMessage instead